pyarr/readarr.py

```diff
@@ -0,0 +1,553 @@
+from .base import BaseAPI
+
+
+class ReadarrAPI(BaseAPI):
+    """API wrapper for Readarr endpoints."""
+
+    def __init__(self, host_url: str, api_key: str):
+        """Initialise Readarr API
+
+        Args:
+            host_url (str): URL for Readarr
+            api_key (str): API key for Readarr
+        """
+
+        ver_uri = "/v1"
+        super().__init__(host_url, api_key, ver_uri)
+
+    def _construct_book_json(
+        self,
+        db_id,
+        book_id_type,
+        root_dir,
+        quality_profile_id=1,
```

Comment:
This is not a safe default. If the very first quality profile created was deleted by the user, then id=1 would no longer exist.

Across all the arrs, we'll need to intelligently handle this scenario for all `*_profile_id` variables.

If `*_profile_id == None` (which should be the default), then Pyarr will need to perform an API query to fetch the first profile ID number.

Reply:
Yes never thought about that, updated to check for the first profile, if no profile exists an exception will raise